TTHAnalysis/python/plotter/susy-sos/scripts/htcondor_runner.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+export XRD_NETWORKSTACK=IPv4
+cd [SRC]
+eval $(scramv1 runtime -sh); 
+cd CMGTools/TTHAnalysis/python/plotter
+[CMD]
+echo "Job for susy-sos task [INST] is done."
+echo "Log files can be found at [WORK]/susy-interface/tmp/[INST]/log"
```

Comment:
Is the `[WORK]` placeholder supposed to be replaced by something in this template? I don't see a relevant substitution in the sos_plots.py script.